Terminal.Gui/Core/Command.cs

```diff
@@ -205,6 +205,41 @@ public enum Command {
 		/// <summary>
 		/// Moves the cursor to the bottom of page.
 		/// </summary>
-		EndOfPage
+		EndOfPage,
+
+		/// <summary>
+		/// Used to do some cleanup-
+		/// </summary>
+		CleanUp,
```

Comment:
I don't think this makes sense as a Command.  Maybe put it back in explicitly as `ProcessKey` code or tie it to a `Leave` event or something?

Reply:
No needed to put it back. I'll test if putting in the `Leave` event works. Thanks.

Reply:
Doesn't work. How about changing it to `CloseItem`?

Reply:
I removed it because the `Toplevel` will process it. Thanks.